src/commands/MakeRoomAdminCommand.ts

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2021 Marco Cirillo
```

Comment:
Also, the copyright should be transferred to the Matrix Foundation. Otherwise, it's by default illegal for us to accept further contributions on this file.

Reply:
Added sign out to commits.